packages/integrations/node/src/server.ts

```diff
@@ -0,0 +1,36 @@
+import type { SSRManifest } from 'astro';
+import type { IncomingMessage, ServerResponse } from 'http';
+import { NodeApp } from 'astro/app/node';
+import { polyfill } from '@astrojs/webapi';
+
+polyfill(globalThis);
```

Comment:
Probably want to make sure `window` and `document` are excluded.

```suggestion
polyfill(globalThis, {
	exclude: 'window document',
});
```

Reply:
Thanks, fixed.